perl/Amanda/Taper/Scribe.pm

```diff
@@ -1255,13 +1255,14 @@ sub _release_reservation {
 	    $label = $self->{'device'}->volume_label();
 	    $fm = $self->{'device'}->file();
 	    $kb = $self->{'device_size'} / 1024;
+	    my $tl = $self->{'taperscan'}->{'tapelist'};
+	    my $tle = $tl->lookup_tapelabel($label);
 
 	    # log a message for amreport
 	    $self->{'feedback'}->scribe_notif_log_info(
-	        message => "tape $label kb $kb fm $fm [OK]");
```

Comment:
As the comment suggests, this log data is parsed by amreport, so I'd expect to see a similar change to amreport to handle this updated format.  Or does this magically appear correctly in the amreport?

Reply:
Hello Dustin,
Answer to your question : Or does this magically appear correctly in the amreport?, is **YES**

The format to read the report is intact with amreport, only the message body is being added with some additional info.
@chassell , If looks good to you , kindly approve the same.